src/main/java/com/direwolf20/buildinggadgets/client/gui/base/EntryList.java

```diff
@@ -0,0 +1,197 @@
+package com.direwolf20.buildinggadgets.client.gui.base;
+
+import com.mojang.blaze3d.platform.GlStateManager;
+import com.mojang.blaze3d.platform.GlStateManager.DestFactor;
+import com.mojang.blaze3d.platform.GlStateManager.SourceFactor;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.AbstractGui;
+import net.minecraft.client.gui.widget.list.ExtendedList;
+import net.minecraft.client.gui.widget.list.ExtendedList.AbstractListEntry;
+import net.minecraft.client.renderer.BufferBuilder;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.util.math.MathHelper;
+import org.lwjgl.opengl.GL11;
+
+import static org.lwjgl.opengl.GL11.*;
+
+/**
+ * A preset of {@link ExtendedList} where
```

Comment:
Can we stay away from the html docs as we're no longer using javadoc for the main source code. No issue with them in the api. You also don't need to justify an implementation in the doc block, simply justify the implementation in your PR.

Reply:
I actually find them quite useful to determine, what this is supposed to do afterwards... Without the comments it's a bit hard to tell what exactly is intended behaviour

Reply:
Comments are fine. The html and go away through haha